content/02.intro.md

```diff
@@ -5,7 +5,7 @@ Machine learning is a modern approach to problem-solving and task automation. In
 Artificial neural networks are a particular class of machine learning algorithms and models that evolved into what we now describe as "deep learning".
 Deep learning encompasses neural networks with many layers and the algorithms that make them perform well.
 These neural networks comprise artificial neurons arranged into layers and are modeled after the human brain, even though the building blocks and learning algorithms may differ [@doi:10.1038/s41583-020-0277-3].
-Each layer receives input from previous layers (the first of which represents the input data) and then transmits a transformed version of its input to the subsequent layer. 
+Each layer receives input from previous layers (the first of which represents the input data), and then transmits its own weighted output that then serves as input into subsequent layers of the network. 
```

Comment:
I happen to like input here (it can be single or plural in this context), but whatever you prefer is totally cool with me :)

Reply:
Yeah I think it is fine then. I just notice that I previously read "layer" as "node" throughout the paragraphs where it would have been more clear to use "inputs" and "output" (since each node has multiple inputs but only 1 output)